src/components/MessageBar/README.md

```diff
@@ -8,7 +8,7 @@ Renders a component that displays relevant status information. You can use a Mes
 | Prop | Type | Description |
 | ---- | ---- | ----------- |
 | ariaLabel * | string | A description of the message bar for the benefit of screen readers. |
-| type | MessageBarType | Dictates the color scheme of the component. Defaults to `MessageBarType.INFO` |
```

Comment:
why?

Reply:
i think `status` better conveys the intent of this prop, `type` is a pretty overloaded term and requires me to look up the definition before i really know what it does.

happy to discuss though. this is just an experimental branch, i mainly wanted to see if i could reduce some CSS by reusing Block and FixedGrid inside MessageBar

Reply:
I prefer type, if anything status implies state to me and they aren't stateful. I wouldnt make this change, especially cos its breaking, but if you want to go through with it and update YamJS I wont put up resistance

Reply:
For what it's worth, Fabric's `MessageBar` also calls this concept `type`, although they use the redundant prop name `messageBarType`: https://developer.microsoft.com/en-us/fabric#/components/messagebar

Reply:
I think I side with @richardtagger, `status` sounds like it's going to change at some point.

Reply:
i can get behind that, i'll revert back to `type`.

actually i think it's the enum name `MessageBarType` that's bugging me. it's really not describing the type of message bar, it's describing the type/context of the message we're displaying -- which in the future could also be used in form errors/warnings, the flash message, maybe others. i'll leave it alone, but i think in the future when we add the FlashMessage component we'll want to decouple that enum from the MessageBar component and name it something like MessageType

Reply:
Yup, sounds reasonable.